lib/Chunk.js

```diff
@@ -265,6 +266,9 @@ class Chunk {
 			}
 			return true;
 		};
+
+		if (otherChunk.isRuntimeOnly) return false;
```

Comment:
>  That would have a different behavior.

You are right, `isOnlyInitial() && hasRuntime()` breaks a test for `limit-chunk-count-plugin`
if set on a limit of one, produces 2 chunks instead of one

I thought of checking `isEmpty()` or `hasEntryModule()`, but the bug is present if there is a named entry with the same name as the `runtimeChunk`

I think `preventIntegrating` is better
but how about `preventIntegration`?

> also check `this.preventIntegration`

Should a Chunk.preventIntegration allow other chunk to be integrated onto itself?
This answers this question, the answer is no.